includes/discovery/vrf.inc.php

```diff
@@ -3,7 +3,8 @@
 use LibreNMS\Config;
 
 if (Config::get('enable_vrfs')) {
-    if ($device['os_group'] == 'cisco' || $device['os'] == 'junos' || $device['os'] == 'ironware') {
+    if ($device['os_group'] == 'vrp' || $device['os_group'] == 'cisco' ||
```

Comment:
Can you update this to:

`if (str_contains($device['os_group'], ['vrp', 'cisco']) || str_contains($device['os'], ['junos', 'ironware'])) {`

Makes it easier to extend these out later.